### DISCUSSION TITLE
버튼을 누를 때마다 버튼 이벤트 부착 함수가 호출됩니다

### DISCUSSION THREAD

**Discussion by Asipe42** [naverz/zepeto-studio-kor#1138] [5. Q&A]
![image](https://user-images.githubusercontent.com/86538738/183322024-354afeee-4e8f-45c7-935d-4b8371d746a5.png)

해당 함수는 Start()에만 넣어 놨는데 해당 함수가 버튼을 누를 때마다 실행됩니다.

잘 작동 되었던 게 에디터 다시 시작하고 나니 작동이 되질 않아요 제페토 플랫폼 문제인 걸까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 버튼 클릭시 어떤 기능을 실행하겠다고 버튼 클릭 이벤트를 연결하는 작업이기 때문에 Start() 안에 선언하는 것입니다.
한번만 실행되길 원하신다면 별도의 isClicked 와 같은 boolean type 변수를 생성하셔서 
실행되도록 구현하시는 것을 추천 드립니다.
